pandas/core/internals/array_manager.py

```diff
@@ -697,6 +699,7 @@ def __init__(
 
         if verify_integrity:
             self._axes = [ensure_index(ax) for ax in axes]
+            arrays = [extract_pandas_array(x, None, 1)[0] for x in arrays]
```

Comment:
Slightly off topic (since we already do this elsewhere), but this check is only needed for when using the patched version in the tests? 
If so, I am wondering if we should use some "TEST_MODE" env variable so we can check that here and do this step only when running the patched test, to avoid the overhead this is adding for real code.

Reply:
yah i increasingly think that test file is more trouble than its worth.  we should try to salvage the subset of tests that directly test PandasArray and disable the others